themes/default/content/blog/deploy-azure-static-website/index.md

```diff
@@ -0,0 +1,205 @@
+```
+
+{{% /choosable %}}
+
+{{% /chooser %}}
+
+There will be some prompts that appear asking you a few questions. Feel free to accept the defaults or change them to your desired preference. This command generated a new project and created a few files for you. You can copy the code examples below for the language you chose to use.
+
+{{< chooser language "typescript,python,csharp,go,yaml" >}}
+
+{{% choosable language typescript %}}
+
+```typescript
+import * as pulumi from "@pulumi/pulumi";
+import * as website from "@pulumi/azure-static-website";
+
+const site = new website.Website("site", {
+   sitePath: "../site", // replace with path containing website contents
+   withCDN: true,
+});
+
+export const { cdnURL } = site;
+```
+
+{{% /choosable %}}
+
+{{% choosable language python %}}
+
+```python
+import pulumi
+import pulumi_azure_static_website as azure_static_website
+
+site = azure_static_website.Website("site",
+    site_path="../site", # replace with path containing website contents
+    with_cdn=True)
+pulumi.export("cdnURL", site.cdn_url)
+```
+
+{{% /choosable %}}
+
+{{% choosable language csharp %}}
+
+```csharp
+using System.Collections.Generic;
+using Pulumi;
+using AzureStaticWebsite = Pulumi.AzureStaticWebsite;
+
+return await Deployment.RunAsync(() =>
+{
+    var web = new AzureStaticWebsite.Website("web", new()
+    {
+        WithCDN = true,
+        SitePath = "../site", // replace with path containing website contents
+    });
+
+    return new Dictionary<string, object?>
+    {
+        ["websiteURL"] = web.CdnURL,
+    };
+});
+```
+
+{{% /choosable %}}
+
+{{% choosable language go %}}
+
+```go
+package main
+
+import (
+	"github.com/pulumi/pulumi-azure-static-website/sdk/go/azure-static-website"
+	"github.com/pulumi/pulumi/sdk/v3/go/pulumi"
+)
+
+func main() {
+	pulumi.Run(func(ctx *pulumi.Context) error {
+		web, err := azure - static - website.NewWebsite(ctx, "web", &azure-static-website.WebsiteArgs{
+			WithCDN:  pulumi.Bool(true),
+			SitePath: pulumi.String("../site"), // replace with path containing website contents
+		})
+		if err != nil {
+			return err
+		}
+		ctx.Export("websiteURL", web.CdnURL)
+		return nil
+	})
+}
+```
+
+{{% /choosable %}}
+
+{{% choosable language yaml %}}
+
+```yaml
+name: az-website
+runtime: yaml
+description: A static website build with pulumi-azure-static-website.
+resources:
+  web:
+    type: "azure-static-website:index:Website"
+    properties:
+      withCDN: true
+      sitePath: "../site"
+outputs:
+  websiteURL: ${web.cdnURL}
+```
+
+{{% /choosable %}}
+
+{{% /chooser %}}
+
+This code creates a new instance of the Website component and sets a couple of properties. `sitePath` is the location where your website contents live and `withCDN` is set to true to let the component know that you would like to provision a CDN. It also exports the `cdnURL` property so you can see where the site can be accessed. That’s all that is needed and now we are ready to deploy!
```

Comment:
I think you should expand a bit on what is expected to be in `sitePath`. For example I think you used React here so for example you could say the component expects the output of a react build, not the path the entire application. I think users might make an assumption the component builds it for them.